packages/zoe/tools/setup-zoe.js

```diff
@@ -61,13 +63,40 @@ export const setUpZoeForTest = async ({
   );
 
   /**
-   * @param {string} path
+   * @param {object} pathOrExports
+   * @returns {Promise<EndoZipBase64Bundle | TestBundle>}
+   */
+  const bundleModule = async pathOrExports => {
+    if (typeof pathOrExports === 'string') {
+      const path = pathOrExports;
+      return bundleSource(path);
+    } else {
+      assert.equal(
+        Object.getOwnPropertyDescriptor(pathOrExports, Symbol.toStringTag)
+          ?.value,
+        'Module',
+      );
+      // Copy all the properties so this object can be hardened.
+      const exports = { ...pathOrExports };
+      return bundleTestExports(exports);
```

Comment:
Does anyone harden the returned bundle?

Reply:
not presently.

https://github.com/endojs/endo/blob/84716b7fb874f790000b96953ef6fe194c4a2f85/packages/import-bundle/src/index.js#L213-L223

If you think that's necessary I'd request that it be done in ^ so the current example code suffices:
https://github.com/endojs/endo/blob/84716b7fb874f790000b96953ef6fe194c4a2f85/packages/import-bundle/README.md?plain=1#L81-L82

Reply:
I think my question is can we get away with dropping the spread if no code would harden the bundle.

If there is such code, then my question for @kriskowal is whether `bundleTestExports` should create a getter for the `Symbol.for('exports')` property instead of a data property so that it's compatible with hardening? That should be a compatible change. It will not make the bundle passable but that is just not possible with this design.